_posts/2022-09-19-public-jenkins.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "OpenSearch automated build system is now live"
+authors: 
+  - sayaligaikawad
+date: 2022-09-14 01:01:01 -0700
+categories: 
+  - community
+---
+
+OpenSearch project deployed automated build, test, and release infrastructure that supports the end-to-end lifecycle of public distributions along with the launch of 2.3.0 version. This includes the migration of OpenSearch Gradle check workflows that are executed on every PR merge, to the public infrastructure. Community users can now view logs by accessing the [build infrastructure](https://build.ci.opensearch.org/) to understand the current state of their workflows and monitor runs as needed.
```

Comment:
```suggestion
OpenSearch project deployed automated build, test, and release infrastructure that supports the end-to-end lifecycle of public distributions along with the launch of 2.3.0 version. This includes the migration of OpenSearch Gradle check workflows that are executed on every PR merge, to the public infrastructure. Community users can now view logs by accessing the [build infrastructure](https://build.ci.opensearch.org/) to understand the current state of their workflows and monitor runs as needed.
```

It looks like some text is missing at the beginning. Add "The" to the beginning of the first sentence.

Reply:
Please include an inline link to the 2.3.0 launch blog here: https://opensearch.org/blog/releases/2022/09/opensearch-2-3-is-ready-for-download/